kitsune/products/views.py

```diff
@@ -44,12 +56,15 @@ def product_landing(request, template, slug):
         else:
             latest_version = 0
 
+    show_community_support = is_localized('get-community-support', request.LANGUAGE_CODE)
```

Comment:
The Past approach was correct. no need to query database if the requested locale is English.
Making another function is overkill